tests/test_dsl.py

```diff
@@ -320,3 +327,141 @@ def assert_cohorts_equivalent(dsl_cohort, qm_cohort):
     # ...and if the columns are the same.
     for k in dsl_col_defs:
         assert repr(dsl_col_defs[k]) == repr(qm_col_defs[k])
+
+
+def test_boolseries_and():
+    series1 = BoolSeries(ValueFromRow(source=None, column="test1"))
+    series2 = BoolSeries(ValueFromRow(source=None, column="test2"))
+
+    output = series1 & series2
+    assert isinstance(output, BoolSeries)
```

Comment:
I think we need to verify the structure of the underlying value rather than just the type here. Sorry.

Reply:
Updated with asserts on the structure of the underlying comparators, was that what you were thinking @benbc?  I'll add them to the other dunder methods if you think that's enough for `BoolSeries.__and__()`.

Reply:
Yes, that's right.

Reply:
@benbc – updated all the series tests with those changes.